calculator_test.go

```diff
@@ -2,23 +2,332 @@ package calculator_test
 
 import (
 	"calculator"
+	"math/rand"
 	"testing"
 )
 
+type testCase struct {
+	a, b        float64
+	want        float64
+	name        string
+	errExpected bool
+}
+
 func TestAdd(t *testing.T) {
```

Comment:
Could we test Add, Subtract, and Multiply in a single test, using the same test case struct for each?